### PR TITLE
Update pr

### PR DESCRIPTION
**Pull request by Ligerx**
Hey it's the dude from reddit.

I've added the kyoto csv from the reddit thread and included an osaka csv that I just scraped.
I've also included links to maps that have imported these csv files.

I don't really use python ever so I messed with the formatting. I also ran into some errors that I fixed, possibly caused by python or library version differences.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
...
-Michelin Star Guide Scraper
+Michelin Star Guide Japan Scraper
 
-- [Tokyo](https://drive.google.com/open?id=1odUZqpAZ02-E6jaKX_6yDAa6qqxoD7N3&usp=sharing)
\ No newline at end of file
+- [Tokyo 2018](https://drive.google.com/open?id=1odUZqpAZ02-E6jaKX_6yDAa6qqxoD7N3&usp=sharing)
+- [Kyoto 2018](https://drive.google.com/open?id=1v3xF-Ue255zZlIEjPQmYpUmRijop2x8D&usp=sharing)
+- [Osaka 2019](https://drive.google.com/open?id=1v3xF-Ue255zZlIEjPQmYpUmRijop2x8D&usp=sharing)
+
+[Reddit thread](https://www.reddit.com/r/JapanTravel/comments/822o3n/michelin_star_map_of_tokyo/)
```
Comment (alexk307) [inline README.md:8]: good idea

File: michelin.py [hunk 2]
```diff
@@ -30,25 +42,29 @@ def _get_rest_info(uri):
...
+		for page in range(1, NUM_PAGES+1):
+			for place in _get_index(page=page):
 				rest_data = _get_rest_info(place)
 				
 				if not header_written:
 					w.writerow(rest_data.keys())
 					header_written = True
+
 				vals = rest_data.values()
-				w.writerow([v.encode('utf-8') for v in vals])
```
Comment (alexk307) [inline michelin.py:50]: I think we still need to encode to `utf-8` here?

Comment (Ligerx) [reply michelin.py:50]: Hmm it's working without the encoding to utf-8.
I removed the binary writing, maybe that makes it not necessary?
Happy to add it back if you want (and I can figure out how to python)

Comment (alexk307) [reply michelin.py:50]: I think we need it just in case there are some strange Kanji characters.

Comment (Ligerx) [reply michelin.py:50]: If I writerow with `[v.encode('utf-8') for v in vals]`, every cell in the resulting csv is wrapped in `b'...'`.

If I instead open the file with these extra params, it outputs without the `b'...'`
`with open(FILE_NAME, 'w', newline='', encoding='utf-8') as f:`

Lemme know which one you want me to go with and I'll push those changes.

Comment (Ligerx) [reply michelin.py:50]: https://stackoverflow.com/a/37364412

Comment (alexk307) [reply michelin.py:50]: Either is fine :)

Thanks for looking into it.

**Comment by Ligerx** [comment]
Added the utf-8 encoding.

I'm leaving tomorrow, so if there's anything else, I won't be able to respond for a couple weeks.

**Comment by alexk307** [comment]
@Ligerx have an amazing trip! Hope you get to go to some of the places on the list :)

**Pull request merged**